packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/TableDemo/TableCompoundExpandableDemo.tsx

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  compoundExpand,
+  IRow,
+  IRowCell
+} from '@patternfly/react-table';
+
+import {
+  CodeBranchIcon,
+  CodeIcon,
+  CubeIcon
+} from '@patternfly/react-icons';
+
+import DemoSortableTable from '@patternfly/react-table/src/components/Table/demo/DemoSortableTable';
+
+export class TableCompoundExpandableDemo extends React.Component<{}, { columns: any, rows: any[] }> {
```

Comment:
Converting from JS to TSX. Receiving error: "You may need an appropriate loader to handle this file type."

Reply:
Yes, we should convert `DemoSortableTable` to TSX locally in this example. Good catch!